modules/serverless-kourier-and-istio-ingresses-overview.adoc

```diff
@@ -36,6 +36,11 @@ See the "Integrating Service Mesh with OpenShift Serverless" section of {Serverl
 [id="serverless-ingresses-traffic-configuration-and-routing_{context}"]
 == Traffic configuration and routing
 
+[NOTE]
+====
+You can find the architecture diagrams for the Kourier and Istio ingresses on the link:https://openshift-knative.github.io/docs/docs/latest/serverless/serving/serving-kourier-istio-ingresses#serverless-ingresses-traffic-configuration-and-routing[Technology and Developer Preview Releases] site.
```

Comment:
I'm not going to hold up a merge review for this if (if it wasn't after hours I would've messaged you about it), but we should probably use "website" instead of "site." "Site" could have translation issues, and generally speaking IBMSG has an entry for website, though it doesn't explicitly say to use this over "site". https://www.ibm.com/docs/en/ibm-style?topic=word-usage#website


```suggestion
You can find the architecture diagrams for the Kourier and Istio ingresses on the link:https://openshift-knative.github.io/docs/docs/latest/serverless/serving/serving-kourier-istio-ingresses#serverless-ingresses-traffic-configuration-and-routing[Technology and Developer Preview Releases] website.
```